src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/TimeValue.txt

```diff
@@ -0,0 +1,52 @@
+﻿>> TimeValue("12:00:00")
+Time(12,0,0,0)
+
+>> TimeValue("27:00:00")
+Time(3,0,0,0)
+
+// TimeValue only returns values between [00:00:00.000, 23:59:59.999)
+>> Value(TimeValue("27:00:00"))
+0.125
+
+>> TimeValue("6:00")
+Time(6,0,0,0)
+
+>> TimeValue("6:00 PM")
+Time(18,0,0,0)
+
+// Date portion is ignored
+>> Value(TimeValue("10/12/2024 6:00:00 AM"))
+0.25
+
+>> TimeValue("29 Feb 2008 9:21:33 AM")
+Time(9,21,33,0)
+
+>> TimeValue("12:34:56.7")
+Time(12,34,56,700)
+
+>> TimeValue("12:34:56.78")
+Time(12,34,56,780)
+
+>> TimeValue("12:34:56.789")
```

Comment:
may be another test with millisecond more than 1000 ?

Reply:
Good idea, added.